mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/WEB-INF/src/com/liferay/todo/TodoPortlet.java

```diff
@@ -0,0 +1,62 @@
+package com.liferay.todo;
+
+import com.liferay.portal.kernel.servlet.SessionErrors;
+import com.liferay.portal.kernel.servlet.SessionMessages;
+import com.liferay.portal.kernel.util.ParamUtil;
+import com.liferay.portal.util.PortalUtil;
+import com.liferay.util.bridges.mvc.MVCPortlet;
+
+import javax.portlet.*;
+import java.io.IOException;
+
+/**
+ * Created by Mateo on 7/13/2017.
+ */
+public class TodoPortlet extends MVCPortlet {
+
+	public void doView(RenderRequest renderRequest, RenderResponse renderResponse)
+
+		throws IOException, PortletException {
+			System.out.println("Render view of Todo Portlet");
+
+			super.doView(renderRequest, renderResponse);
+	}
+
+	public void formAction(ActionRequest actionRequest, ActionResponse actionResponse)
+			throws IOException, PortletException {
+			try {
```

Comment:
Don't wrap everything in ```try / catch```, unless you have different error types handling in ```catch``` block. Liferay already has catch-all runtime exceptions handling that ensures page does not break, but you get an error message in UI.